rhp/v4/server.go

```diff
@@ -253,7 +252,7 @@ func (s *Server) handleRPCWriteSector(stream net.Conn) error {
 		return errorDecodingError("failed to read sector data: %v", err)
 	}
 
-	usage := prices.RPCWriteSectorCost(req.DataLength, req.Duration)
+	usage := prices.RPCWriteSectorCost(req.DataLength)
```

Comment:
This is slightly odd to me. The `RPCWriteSectorCost` which is in `core` uses `TempSectorDuration` now but the `RPCWriteSectorRequest` which is also declared in `core`, still allows for specifying a `duration`. 
Is that a leftover or do we want to keep the protocol flexible even if we expect to always use `TempSectorDuration` @n8maninger ?

Reply:
That is left over and should be removed